### PR TITLE
lucie's changes - payment & pricing html skeletons

### PR DESCRIPTION
**Pull request by luciebrock**
please check app/views/sessions/pricing.html.erb & app/views/sessions/payment.html.erb for errors (my mysql is broken)

### CODE REVIEW

File: app/views/sessions/pricing.html.erb [hunk 1]
```diff
@@ -1 +1,25 @@
+<br/>
+<img src="/assets/logo.png" alt = "Background">
+<br/>
+<br/>
+
+
 <h1>Pricing</h1>
+#do i need to make this h2 lol
```
Comment (epartakki) [inline app/views/sessions/pricing.html.erb:8]: up to you! we can figure it out next week when we figure out how the website will look across the board.

File: app/views/sessions/pricing.html.erb [hunk 2]
```diff
@@ -1 +1,25 @@
...
+<%= button_to "Get Day Pass", '/payment', method: :get%>
+<br/>
+#<p> Month pass, $20/month, unlimited 45 minute rides for a month </p>
+<%= button_to "Get Month Pass", '/payment', method: :get%>
+<br/>
+#<p>Annual pass, $80/year, unlimited 45 minute rides for a year</p>
+<%= button_to "Get Annual Pass", '/payment', method: :get%>
+<br/>
+
+#other options or nah?
```
Comment (epartakki) [inline app/views/sessions/pricing.html.erb:25]: looks great for now!

File: app/views/sessions/payment.html.erb [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<br/>
+<img src="/assets/logo.png" alt = "Background">
+<br/>
+<br/>
+
```
Comment (epartakki) [inline app/views/sessions/payment.html.erb:5]: can you add a url at the top from where you got some of this code?

**Review by epartakki — APPROVED** [review]
Just make sure to add citations so we can go back and find solutions when we need to! Great start Lucie!

**Review by sunniraleigh — APPROVED** [review]
looks good

File: app/views/sessions/pricing.html.erb [hunk 4]
```diff
@@ -1 +1,25 @@
...
+<img src="/assets/logo.png" alt = "Background">
+<br/>
+<br/>
+
+
 <h1>Pricing</h1>
+#do i need to make this h2 lol
+<h3>Select your plan.<h3>
+
+#none of these are funcitonal
```
Comment (amarrap) [inline app/views/sessions/pricing.html.erb:11]: funcitonal

**Review by amarrap — APPROVED** [review]
nice

File: app/views/sessions/payment.html.erb (hunk 3 above)
Comment (luciebrock) [reply app/views/sessions/payment.html.erb:5]: I literally got this code from the welcome page you made lol

File: app/views/sessions/pricing.html.erb (hunk 4 above)
Comment (luciebrock) [reply app/views/sessions/pricing.html.erb:11]: in some ways

Comment (luciebrock) [reply app/views/sessions/pricing.html.erb:11]: bye

File: app/views/sessions/pricing.html.erb (hunk 1 above)
Comment (luciebrock) [reply app/views/sessions/pricing.html.erb:8]: you are the citation and the other code is from https://www.paypal.com/buttons/smart

File: app/views/sessions/payment.html.erb [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+<h1>Payment</h1>
+#do i need to make this h2 lol
```
Comment (kvangreen) [inline app/views/sessions/payment.html.erb:8]: Hashtag comments don't work on html.erb files, as we discovered last Thursday

File: app/views/sessions/payment.html.erb [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    var quantitySelect = document.querySelector("#smart-button-container #quantitySelect");
+    if (!isNaN(quantity)) {
+      quantitySelect.style.visibility = "visible";
+    }
+    var orderDescription = 'ValetBike Ride';
+    if(orderDescription === '') {
+      orderDescription = 'Item';
+    }
+    paypal.Buttons({
+      style: {
```
Comment (kvangreen) [inline app/views/sessions/payment.html.erb:36]: Could we pull this into a CSS file later?

**Review by kvangreen — APPROVED** [review]
Sexy like you

File: app/views/sessions/payment.html.erb (hunk 5 above)
Comment (kvangreen) [reply app/views/sessions/payment.html.erb:8]: NEVER MIND I AM A TRAGIC CLOWN APPARENTLY

**Pull request merged**